src/index.css

```diff
@@ -0,0 +1,14 @@
+@import "./blocks/page.css";
+@import "./blocks/stars.css";
+@import "./blocks/card.css";
+
+@font-face {
+  font-family: "Roboto-regular";
+  src: url(./fonts/Roboto-Regular.ttf);
+}
+
+@font-face {
+  font-family: "Roboto-medium";
+  src: url(./fonts/Roboto-Medium.ttf);
+  font-style: normal;
+}
```

Comment:
https://user-images.githubusercontent.com/94289038/215411674-30c13ced-1223-411d-89e1-1f6269e3195a.mov

yes - in devTools all looks ok
you used 700 font-weight, but you didn't import it - so it is not applied 
fix it pls (add new rule here or use font links in index HTML)
ask help of mentors in `fe_chat` if you need it

Reply:
Thank you!
Now its clear to me.  Font issue fixed (Roboto bold 700 was missing). 
Also picture moved from <div> background to <img>
which changed its size within few pixels.